_schemaV2.graphql

```diff
@@ -2163,20 +2163,16 @@ type ArtistRelatedData {
   ): ArtistConnection
 }
 
-type ArtistSeries {
-  # List of Mongo IDs for associated Artists
+type ArtistSeries implements Node {
```

Comment:
cc @gkartalis this PR unstitches ArtistSeries and adds this support - I _think_ this, once it lands shortly - should help the issue you were discussing.